.travis.yml

```diff
@@ -2,7 +2,7 @@
 before_install:
  - sudo add-apt-repository -y ppa:codegear/release
  - sudo apt-get -qq update
- - sudo apt-get install premake4
+ - sudo apt-get install premake4 uuid-dev
```

Comment:
Are there also lib packages present for mingw (cross-compiling for windows)? And how to source it for OSX? BSD?
I'm not really good with this unless it addresses the build for Windows, OSX, BSD, etc aswell.